src/Tools/CMakeLists.txt

```diff
@@ -13,6 +13,6 @@ endmacro(TOOL)
 
 TOOL(ConvertPointCloud      ${CMAKE_PROJECT_NAME})
 TOOL(EncodeShader)
-TOOL(ManuallyCropPointCloud ${CMAKE_PROJECT_NAME})
+TOOL(ManuallyCropGeometry   ${CMAKE_PROJECT_NAME})
```

Comment:
I changed the application name. Do we have any dependent projects regarding this program names?

Reply:
I think this is fine.